string satellite_id
uint32 packet_time_size
WOD_data[] datasets
